public/content/documentation/web/component/modal-dialog.md

```diff
@@ -198,8 +198,41 @@ Some browsers require additional scripting. This simple example works in Chrome,
 
 ## Videos
 
-- Videos go here
+### Android Talkback
+
+<video controls>
+  <source src="media/video/web/modal-dialog/modalDialog_AndroidTalkback.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+
```

Comment:
Known issue, I'll remove the extra row when I update the TODO needed in the code example.